src/app/pages/loading-states/loading-states.component.html

```diff
@@ -0,0 +1,54 @@
+<mat-toolbar color="primary" flexLayout="row">
+    <button *ngIf="isSmall" mat-icon-button (click)="toggleMenu()" style="margin-left: -8px;">
+        <mat-icon>menu</mat-icon>
+    </button>
+    <h6>Loading States</h6>
+    <div fxFlex></div>
+    <button mat-button color="white" (click)="refreshData()">
+        <mat-icon>autorenew</mat-icon>
+    </button>
+</mat-toolbar>
+
+<div style="padding: 20px;" class="loading-states-container">
```

Comment:
should we do 16, or 24 so it's a multiple of the base unit?

Reply:
Yep, went with 24